packages/eslint-plugin/docs/rules/no-unused-expressions.mdx

```diff
@@ -10,4 +10,8 @@ import TabItem from '@theme/TabItem';
 > See **https://typescript-eslint.io/rules/no-unused-expressions** for documentation.
 
 This rule extends the base [`eslint/no-unused-expressions`](https://eslint.org/docs/rules/no-unused-expressions) rule.
-It adds support for optional call expressions `x?.()`, and directive in module declarations.
+It supports TypeScript-specific expressions:
+
+- Handles instantiation expressions: `Set<number>;`
```

Comment:
what does this mean?

Reply:
This is an expression by itself.

```ts
Set<number>;
```

Reply:
Ah. I'm not sure that "instantiation expressions" is meaningful to a user, but it's probably not terribly important. Maybe this would be easier to follow as simply a "correct"/"incorrect" block, with more vague wording? Again not terribly important though, I defer to you on what you like best 👍